stateless.go

```diff
@@ -78,9 +78,7 @@ func NewStateless() *StatelessNode {
 }
```

Comment:
I accommodated things in this file to make it compile, but it won't work.
I didn't care much since @gballet mentioned that we'll kill all this anyway.